go/serialization_test.go

```diff
@@ -430,19 +430,19 @@ func TestUnmarshalDigitallySigned(t *testing.T) {
 }
 
 func TestSCTSerializationRoundTrip(t *testing.T) {
```

Comment:
Again, these tests were to test the Serialize/DeserializeSCT functions, so maybe are no longer needed now that all they seem to be testing are the tls Marshal/Unmarshal functions?  Up to you.  But once again, if you do choose to keep them, change the error messages to reflect that it is marshalling/unmarshalling that is failing.